src/service/telemetryHandlerService.ts

```diff
@@ -59,6 +69,8 @@ export class TelemetryHandlerService {
             case CONSTANTS.DEVICE_NAME.MICROBIT:
                 this.handleMicrobitButtonPressTelemetry(buttonState);
                 break;
+            case CONSTANTS.DEVICE_NAME.CLUE:
+                this.handleClueButtonPressTelemetry(buttonState)
```

Comment:
Perhaps add a break before the default statement, in case we add more cases in the future.

Reply:
good catch!